app/src/androidTest/java/com/android/sample/ui/activityDetails/CommentItemTest.kt

```diff
@@ -62,7 +66,8 @@ class CommentItemTest {
           profileId = nonCreatorProfileId,
           comment = testComment,
           onReplyComment = { _, _ -> },
```

Comment:
Adding a creatorId parameter to CommentItem and related test cases to identify the comment's creator.